release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -1594,3 +1594,51 @@ This is because the service CA is automatically rotated as of {product-title} 4.
 After this one-time manual restart, subsequent upgrades and rotations will ensure restart before the service CA expires without requiring manual intervention.
 ====
 ////
+
+[[ocp-4-3-2]]
+=== RHBA-2020:0491 - {product-title} 4.3.2 Bug Fix Update
+
+Issued: 2020-02-17
+
+{product-title} release 4.3.2 is now available. The list of packages included in
+the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:0491[RHBA-2020:0491] advisory.
+The container images and bug fixes included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:0492[RHBA-2020:0492] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/4828131[{product-title} 4.3.2 container image list]
+
+[[ocp-4-3-2-bug-fixes]]
+==== Bug Fixes
```

Comment:
This is great and a totally valid option - typically we only put bug summaries in the release notes if they do not fit in the errata character limit of 4000 characters. Otherwise errata look something like this: https://access.redhat.com/errata/RHBA-2019:2352 

I'm not certain if these bug summaries cross that character limit or not, but just a reminder anyway :) Looks great, Cody!

Reply:
Good call! I honestly was so used to the bug summaries eclipsing the character limit that I was adding them to RNs without checking. I'll make a note of this and check moving forward. Thanks!